src/alpaka_job_coverage/filter_software_dependency.py

```diff
@@ -104,6 +135,7 @@ def software_dependency_filter(row: List) -> bool:
             )
         )
     ):
+        reason(output, "clang 11 and 12 are not available in the Ubuntu 18.04 ppa")
```

Comment:
```suggestion
        reason(output, "clang 11 and 12 are not available in the Ubuntu 18.04 PPA")
```

Also Ubuntu 18.04 is no longer in "standard support" - do we still care about this release?